Rubberduck.VBEditor.VBA/SafeComWrappers/VB/Windows.cs

```diff
@@ -35,7 +35,7 @@ public ToolWindowInfo CreateToolWindow(IAddIn addInInst, string progId, string c
             return new ToolWindowInfo(new Window(window), control);
         }
 
-        public static void ReleaseDockableHosts()
+        public void ReleaseDockableHosts()
```

Comment:
Can you elaborate the reasoning why this was made into an instance method?

Reply:
Used in the shutdown process in Extension.cs in Rubberduck.Main. As a static method, this needed a hard reference to the (VBA) SCW assembly. As an interface method, it can be exposed on the IWindow interface, removing the hard dependency.